companion/src/modelprinter.cpp

```diff
@@ -228,13 +228,14 @@ QString ModelPrinter::printMultiSubType(unsigned rfProtocol, bool custom, unsign
 
 QString ModelPrinter::printR9MPowerValue(unsigned subType, unsigned val, bool telem)
 {
+  #warning "Here R9M power values are wrong";
```

Comment:
VC++ compilation fails on the #warning...

Reply:
I will review the whole section as it is wrong for xlite